src/products/pages/CreateProductPage.js

```diff
@@ -55,7 +55,7 @@ export const CreateProductPage = props => {
         <Header as="h2" color="teal" textAlign="center">
           Create your product
         </Header>
-        <div>
+        <div data-cy="error-message">
```

Comment:
This `div` iterates though an array or error messages:
```suggestion
        <div data-cy="error-messages">
```